version: "3.1"
services:
  jupyterlab:
    image: jupyter/datascience-notebook:latest
    container_name: jupyterlab
    user: root
    ports:
      - "8888:8888"
    restart: unless-stopped
    volumes:
      - ${DOCKERDATA}/jupyterlab:/home/${USERNAME}/work
    environment:
      - NB_USER=${USERNAME}
      - NB_UID=1000
      - NB_GID=1000
      - NB_GROUP=${USERNAME}
      - CHOWN_HOME=yes
      - GRANT_SUDO=yes
      - RESTARTABLE=yes
    working_dir: /home/${USERNAME}/work
